shenyu-spring-boot-starter/shenyu-spring-boot-starter-plugin/shenyu-spring-boot-starter-plugin-oauth2/src/main/java/org/apache/shenyu/springboot/starter/plugin/oauth2/OAuth2PluginConfiguration.java

```diff
@@ -0,0 +1,164 @@
+/*
+ *   Licensed to the Apache Software Foundation (ASF) under one or more
+ *   contributor license agreements.  See the NOTICE file distributed with
+ *   this work for additional information regarding copyright ownership.
+ *   The ASF licenses this file to You under the Apache License, Version 2.0
+ *   (the "License"); you may not use this file except in compliance with
+ *   the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
```

Comment:
Please reformat the license header.  see [https://github.com/apache/incubator-shenyu/blob/068c984678978e22136e134d12dec54b43f453f5/shenyu-plugin/shenyu-plugin-oauth2/src/test/java/org/apache/shenyu/plugin/oauth2/filter/OAuth2PreFilterTest.java](https://github.com/apache/incubator-shenyu/blob/068c984678978e22136e134d12dec54b43f453f5/shenyu-plugin/shenyu-plugin-oauth2/src/test/java/org/apache/shenyu/plugin/oauth2/filter/OAuth2PreFilterTest.java)

Reply:
ok,i had change it.